lua/ttt2/libraries/plyspawn.lua

```diff
@@ -203,7 +203,7 @@ function plyspawn.GetRandomSafePlayerSpawnPoint(ply)
 
 		if not riggedSpawnPoint then continue end
 
-		ErrorNoHalt("TTT2 WARNING: Map has too few spawn points, using a riggedSpawnPoints spawn for " .. tostring(ply) .. "\n")
+		ErrorNoHaltWithStack("TTT2 WARNING: Map has too few spawn points, using a riggedSpawnPoints spawn for " .. tostring(ply) .. "\n")
```

Comment:
This is on the map maker and happens quite a lot. Please remove that here, there is nothing the dev can change about that